frontend/pages/index.tsx

```diff
@@ -1,11 +1,20 @@
 import { NextPage } from 'next'
+
+import { withApollo } from '../lib/apollo'
+
 import LandingPageLayout from '../components/Layouts/LandingPageLayout'
 import Home from '../components/Site/Home'
 
-const HomePage: NextPage = () => (
-  <LandingPageLayout>
-    <Home />
-  </LandingPageLayout>
-)
+interface InitialProps {}
```

Comment:
Ah I meant to leave a comment about this. I included these as a reference because we'll need them when we port over the i18n internationalization code where we are getting initial props and so on 😊